tests/ocpp-server/server.py

```diff
@@ -0,0 +1,89 @@
+import asyncio
+import logging
+import websockets
+from datetime import datetime, timezone
+
+from ocpp.routing import on
+from ocpp.v201 import ChargePoint as cp
+from ocpp.v201 import call_result
+from ocpp.v201.enums import RegistrationStatusType, GenericDeviceModelStatusType
+
+# Setting up the logging configuration to display debug level messages.
+logging.basicConfig(level=logging.DEBUG)
+
+# Define a ChargePoint class inheriting from the OCPP 2.0.1 ChargePoint class.
+class ChargePoint(cp):
+    # Define a handler for the BootNotification message.
+    @on('BootNotification')
```

Comment:
More "stub" OCPP command handlers will be needed to allow the simulator to work. 
Have you tested it with the core in the PR?